cloud-optimized-geotiffs/writing-cogs-in-python.ipynb

```diff
@@ -0,0 +1,452 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    # "zip:///Users/rajatshinde/impact/cloud-optimized-geospatial-formats-guide/cloud-optimized-geotiffs/data/NASADEM_HGT_n57e105.zip!n57e105.hgt"
</pre><p>Does this work with a relative path? The would be preferred for the tutorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>